lib/services/destinychat.js

```diff
@@ -67,6 +67,9 @@ class DestinyChat extends EventEmitter {
       }
     }
 
+    if (_.startsWith(event.data, 'ERR')) {
+      this.emit('err', event.data.replace('ERR ', ''));
+    }
```

Comment:
Let's get rid of the comment for `parseMessages()` considering we now care about messages that aren't from users.